### ISSUE TITLE
希望添加一个在放大之后可以拖动图片的功能

### ISSUE THREAD

**Issue by MOON1234567890A** [CVHub520/X-AnyLabeling#751]
### Search before asking

- [X] I have searched the X-AnyLabeling [issues](https://github.com/CVHub520/X-AnyLabeling/issues) and found no similar feature requests.

### Description

你好，希望添加一个在放大之后可以拖动图片的功能，鼠标滚动只能滚动上下，对一些需要左右连续标注的的不是很友好，同时拖动下面的条容易略过小物体，我发现在  #729 第二条里面有相关的回答，但是我试用了效果并不是很好，切换只能应用在检查的时候
![image](https://github.com/user-attachments/assets/8d1616ca-2fa9-47f6-8557-ec56ae9fcad0)

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by CVHub520** [comment]
You can hold `Ctrl` + scroll the mouse wheel to zoom in or out around the current cursor position.

**Comment by MOON1234567890A** [comment]
Yes, but if I need to label adjacent but equally small objects, I have to either drag the lower bar (causing me to pull too much at once and miss the small object) or zoom out and then zoom in (tedious steps)
(╥_╥)

**Comment by CVHub520** [comment]
Oh, I see. I'm so sorry about that. Unfortunately, I currently lack the time and energy to implement this interactive feature and will have to rely on support from the community for now. :-(

**Comment by lhj5426** [comment]
我也想要个PS 那样的 按住 空格可以方便拖拽 最好再有个小窗口的全图预览
不过 可能是就 想法了  
但是关于放大后的左右滚动我是这么解决的
![image](https://github.com/user-attachments/assets/c043166e-11ca-4a26-9a8e-ce057a8b0bf1)
用 鼠标手势 来 模拟鼠标的左右滚动
这样就保证了 打标的左手在 打标键盘上 不需要因为放大缩小而来回移动到 主键盘按CTRL

**Comment by lhj5426** [comment]
另外 他的顶点缩放 并不是以当前鼠标所在的点为中心放大 而是 在周围放大  
放大我用了 Glassbrick_v2.3 这个 工具来替代
![image](https://github.com/user-attachments/assets/eedaa61b-829b-4036-a085-2eefd8df04ac)

这个是真的以当前 鼠标所在的点 为中心进行放大的
软件的交互还有很多不足 不过用这些工具勉强能补充支持

**Comment by CVHub520** [comment]
Hi there! 👋 Thanks for reaching out. We're glad to inform you that the functionality to drag the canvas with your mouse after zooming in is already available in the latest version of X-AnyLabeling. For detailed instructions on how to use this feature, please refer to our user manual docs. We hope this helps improve your workflow! 😊
